lookout/style/typos/analyzer.py

```diff
@@ -153,9 +154,9 @@ def filter_suggestions(self, test_df: pandas.DataFrame,
         Filter suggestions based on the repo specifics and confidence threshold.
 
         :param test_df: DataFrame with info about tested tokens.
-        :param suggestions: Dictionary of correction suggestions grouped by \
+        :param suggestions: Dictionary of correction suggestions grouped by
```

Comment:
I think you should use ` \` because otherwise, param docsting will end here. But I am not 100% sure. Can you check it or read in doc?

Reply:
Yes, it's a mistake.